Makefile

```diff
@@ -0,0 +1,52 @@
+default: 
+	@echo "Call a specific subcommand:"
+	@echo
+	@$(MAKE) -pRrq -f $(lastword $(MAKEFILE_LIST)) : 2>/dev/null\
+        | awk -v RS= -F: '/^# File/,/^# Finished Make data base/ {if ($$1 !~ "^[#.]") {print $$1}}'\
+        | sort\
+        | egrep -v -e '^[^[:alnum:]]' -e '^$@$$'
+	@echo
+	@exit 1
+
+.state/docker-build-web: Dockerfile dev-requirements.txt base-requirements.txt
+	#Build web container for this project 
+	docker-compose build --force-rm web
+
+	#Mark the state so we don't rebuild this needlessly.
+	mkdir -p .state && touch .state/docker-build-web
+
+.state/db-migrated:
+	#Call migrate target
+	make migrate 
+
+	#Mark the state so we don't rebuild this needlessly.
+	mkdir -p .state && touch .state/db-migrated
+
+.state/db-initialized: .state/docker-build-web .state/db-migrated
+	#Load all fixtures 
+	docker-compose run --rm web ./manage.py loaddata fixtures/*.json
+
+	#Mark the state so we don't rebuild this needlessly.
+	mkdir -p .state && touch .state/db-initialized
+
+serve: .state/db-initialized
+	docker-compose up --remove-orphans
+
+migrations: .state/docker-build-web
```

Comment:
We want an initialized database before being able to create new migrations, so this should have `.state/db-initialized` as a requisite, not just a built docker image.